frontend/app/routes/_app+/letters+/_index.tsx

```diff
@@ -74,12 +74,12 @@ export default function LettersIndex() {
       </div>
       <ul className="divide-y border-y">
         {letters.map((letter) => {
-          const letterType = letterTypes.find(({ code }) => code === letter.name);
-          const letterName = letterType ? letterType[i18n.language === 'fr' ? 'nameFr' : 'nameEn'] : letter.name;
+          const letterTypeCd = letterTypes.find((letterType) => letterType.code == letter.name);
+          const letterTypeCode = { nameEn: letterTypeCd?.nameEn, nameFr: letterTypeCd?.nameFr };
```

Comment:
Try to find a macthin letter type with the letter's name and type's code. If it's undefined then fallback with the letter's name.
```suggestion
          const letterType = letterTypes.find(({code}) => code == letter.name);
          const letterName = letterType ? getNameByLanguage(i18n.language, letterType) : letter.name;
```

Reply:
Fixed: 04e0b73a6520ee0dc46d6b68ece420e1df40cca8

Whats the difference in line 77?

Reply:
Fixed: https://github.com/DTS-STN/canadian-dental-care-plan/commit/04e0b73a6520ee0dc46d6b68ece420e1df40cca8, https://github.com/DTS-STN/canadian-dental-care-plan/commit/e59829483ec0cbe75199f06d5638bb9fcdb81887